sass/base/address.scss

```
@@ -0,0 +1,167 @@
+.detail-show__nav {
+	@include showNav;
+}
+
+.address {
+	padding: 0 10%;
+	margin-bottom: 90px;
+	position: relative;
+
+	.main-intro__tittle {
+		margin-bottom: 30px;
+
+		h1 {
+			font-size: 23px;
+			padding-bottom: 2px;
+		}
+	}
+
+	&-form {
+		margin-bottom: 25px;
+		@include flexDisplay(row);
+		@include maxWidth(900px) {
+			@include flexDisplay(column);
+		}
+
+		&__content {
+			color: rgba($colorBlack, 0.8);
+			flex: 50%;
+			max-width: 50%;
+			@include maxWidth(900px) {
+				flex: 100%;
+				max-width: 100%;
+				margin: 0;
+			}
+
+			&:first-child {
+				margin-right: 15px;
+			}
+
+			&:last-child {
+				margin-left: 15px;
+			}
+
+			h3 {
+				text-transform: uppercase;
+			}
+
+			a {
+				color: $colorBlue;
+				font-size: 17px;
+			}
+
+			form {
+				margin-top: 30px;
+
+				p {
+					margin-bottom: 15px;
+					@include flexDisplay(row);
+
+					label {
+						flex: 20%;
+						max-width: 20%;
+						display: block;
+					}
+
+					input {
+						flex: 80%;
+						max-width: 80%;
+						width: 100%;
+						display: block;
+						border: 1px solid rgba($colorBlack, 0.2);
+						padding: 7px;
+					}
+
+					select {
+						flex: 80%;
+						max-width: 80%;
+						display: block;
+						border: 1px solid rgba($colorBlack, 0.2);
```

rgba không nhận giá trị $colorBlack: black theo cách này đâu ạ
```suggestion
							background: rgba(0, 0, 0, 0.2);
```

---

Sao ko đc em nhỉ? Chì thấy ở đây họ có chỉ làm như thế nè https://sass-lang.com/documentation/variables

---

vẫn dùng được colorBlack nha c, scss, sass nó tự convert